lxd/certificates.go

```diff
@@ -57,7 +58,27 @@ func (d *Daemon) verifyAdminPwd(password string) bool {
 	return true
 }
 
+type certificateResponse struct {
```

Comment:
Yeah, though it's not used by any client so it's not a big deal.

Actually this sort of makes it spec compliant as the previous implementation used "certificates" rather than "certificate" as defined in the spec.

The only change compared to the current specification is the addition of the fingerprint field, which you should actually add to specs/rest-api.md too so the spec is accurate.